lib/widgets/compose_box.dart

```diff
@@ -89,13 +89,15 @@ enum TopicValidationError {
 }
 
 class ComposeTopicController extends ComposeController<TopicValidationError> {
-  ComposeTopicController() {
+  ComposeTopicController({required this.store}) {
     _update();
   }
 
+  PerAccountStore store;
+
   // TODO: subscribe to this value:
   //   https://zulip.com/help/require-topics
```

Comment:
What's the condition for this existing TODO to be finished, and can we be more concrete about it? It looks like it's meant to be a `TODO(#668)`, right? Does `ComposeTopicController` need to add a `PerAccountStore` listener?

Reply:
> Does ComposeTopicController need to add a PerAccountStore listener?

Right. We may include that as a part of #668 or its follow-up. This would be something for the future us to triage. I will update to mention the issue.